bst.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var recursion = function(root, min, max) {
+    if (!root) return true;
+    return root.val < max && root.val > min
+        ? recursion(root.left, min, root.val) && recursion(root.right, root.val, max)
+        : false;
+};
+
+var isValidBST = function(root) {
+    if (!root || (!root.left && !root.right)) return true;
```

Comment:
This will work the same
```suggestion
    if (!root) return true;
```

Reply:
[0], [1] - edge cases

Reply:
What do you mean?